src/pages/InsideContents/ChatAiPage.tsx

```
@@ -0,0 +1,262 @@
+                        ? theme.palette.primary.main 
+                        : theme.palette.background.paper,
+                      color: msg.type === 'user' 
+                        ? theme.palette.primary.contrastText 
+                        : theme.palette.text.primary,
+                      borderRadius: msg.type === 'user' 
+                        ? '15px 15px 5px 15px' 
+                        : '15px 15px 15px 5px',
+                    }}
+                  >
+                    {msg.type === 'ai' ? (
+                      <div dangerouslySetInnerHTML={{ __html: msg.content.replace(/\n/g, '<br>') }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)